docs/openj9_support.md

```diff
@@ -66,60 +65,147 @@ For any issues or limitations of an Eclipse OpenJ9 release, read the [release no
+| Centos 6                              |   Y    |   Y    |     Y      |   N  |  N   |
+| Centos 7                              |   Y    |   Y    |     Y      |   N  |  N   |
+| Red Hat Enterprise Linux (RHEL) 6     |   Y    |   Y    |     Y      |   Y  |  Y   |
+| RHEL 7                                |   Y    |   Y    |     Y      |   Y  |  Y   |
+| SUSE Linux Enterprise Server (SLES) 12|   Y    |   Y    |     Y      |   Y  |  Y   |
+| Ubuntu 16.04                          |   Y    |   Y    |     Y      |   N  |  Y   |
+| Ubuntu 18.04                          |   Y    |   Y    |     Y      |   N  |  Y   |
+
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i><span class="sr-only">Note</span> **Note:** Not all of these distributions are tested, but Linux distributions that have a
+minimum glibc version 2.12 are expected to function without problems.
+
+| Windows                    |  x32   |  x64  |
+|----------------------------|--------|-------|
+| Windows 10                 |   Y    |   Y   |
+| Windows Server 2012        |   Y    |   Y   |
+| Windows Server 2012 R2     |   Y    |   Y   |
+| Windows Server 2016        |   Y    |   Y   |
+
+
+| AIX          |  ppc32   | ppc64    |
+|--------------|----------|----------|
+| AIX 7.1 TL4  |    Y     |    Y     |
+| AIX 7.2      |    Y     |    Y     |
+
+When public support for an operating system version ends, OpenJ9 can no longer be supported on that level.
+
 
 
-OpenJDK9 binaries are supported on the minimum operating system levels shown in the table.
-For Linux, other distributions are supported provided that they have a Linux kernel v4.4 or later
-(Look in the `/proc/version` file to find the kernel version number).
+### OpenJDK 10
+
+OpenJDK10 binaries are supported on the minimum operating system levels shown in the following tables:
+
+
+| Linux                                 | x32    |  x64   |  ppc64le   | Z31  | Z64  |
+|---------------------------------------|--------|--------|------------|------|------|
+| Centos 6                              |   Y    |   Y    |     Y      |   N  |  N   |
+| Centos 7                              |   Y    |   Y    |     Y      |   N  |  N   |
+| Red Hat Enterprise Linux (RHEL) 6     |   Y    |   Y    |     Y      |   Y  |  Y   |
+| RHEL 7                                |   Y    |   Y    |     Y      |   Y  |  Y   |
+| SUSE Linux Enterprise Server (SLES) 12|   Y    |   Y    |     Y      |   Y  |  Y   |
+| Ubuntu 16.04                          |   Y    |   Y    |     Y      |   N  |  Y   |
+| Ubuntu 18.04                          |   Y    |   Y    |     Y      |   N  |  Y   |
+
+<i class="fa fa-pencil-square-o" aria-hidden="true"></i><span class="sr-only">Note</span> **Note:** Not all of these distributions are tested, but Linux distributions that have a
+minimum glibc version 2.12 are expected to function without problems.
+
+| Windows                    |  x32   |  x64   |
+|----------------------------|--------|--------|
```

Comment:
32 bit windows was only intended to be supported for JDK8.  No effort has been put into making it function for new JDK levels

Reply:
Done